source/draw/Color.ooc

```diff
@@ -227,16 +160,9 @@ ColorBgra: cover {
 	toYuva: func -> ColorYuva { ColorYuva new(this bgr toYuv(), this alpha) }
 	toBgr: func -> ColorBgr { this bgr copy() }
 	toBgra: func -> This { this copy() }
-	blend: func (factor: Float, other: This) -> This {
-		This new(this bgr blend(factor, other bgr), (this alpha * (1 - factor) + other alpha * factor) as Byte)
-	}
-	distance: func (other: This) -> Float {
-		(this bgr distance(other bgr) * 3.0f + (this alpha - other alpha) as Float pow(2)) / 4.0f sqrt()
-	}
-	equals: func ~monochrome (other: ColorMonochrome) -> Bool { false }
-	equals: func ~yuv (other: ColorYuv) -> Bool { false }
-	equals: func ~bgr (other: ColorBgr) -> Bool { false }
 	equals: func ~bgra (other: This) -> Bool { this bgr equals(other bgr) && this alpha == other alpha }
+	blend: func (factor: Float, other: This) -> This { This new(this bgr blend(factor, other bgr), (this alpha * (1 - factor) + other alpha * factor) as Byte) }
+	distance: func (other: This) -> Float { (this bgr distance(other bgr) * 3.0f + (this alpha - other alpha) as Float pow(2)) / 4.0f sqrt() }
```

Comment:
The bgr distance divided by 3 before taking the square root so it would be `|bgr|^2 * 3` to get the square distance but it is better to just redo both into the real distance in 4 dimensional space unless something depends on the weirdness. `sqrt(b^2 + g^2 + r^2 + a^2)`